client/app/nasha/add/nasha-unavailable.html

```diff
@@ -0,0 +1,31 @@
+<div class="cui-page">
+    <oui-page-header
+        data-heading="{{::'nasha_order_title' | translate}}">
+    </oui-page-header>
+
+    <div class="cui-page__content nasha-unavailable">
+        <h2 class="oui-header_3" data-translate="nasha_order_unavailable_subtitle"></h3>
+
+        <div class="nasha-unavailable__contact">
```

Comment:
Since we are able to use some helper classes and avoid custom styles as much as we can, what do you think about:

```html
<div class="d-flex align-items-center">
    <div class="m-2">
        <svg width="32" height="32" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32">
            …
        </svg>
    </div>
    <div class="pl-3">
        <div data-translate="nasha_order_unavailable_explain"></div>
        <strong data-ng-bind="$ctrl.phone"></strong>
    </div>
</div>
```

In that way you can delete `client/app/nasha/add/nasha-unavailable.less`.